sickbeard/showUpdater.py

```diff
@@ -118,11 +119,10 @@ def run(self, force=False):  # pylint: disable=unused-argument, too-many-locals,
                             logger.log(u"Unable to update show: {0}".format(str(e)), logger.DEBUG)
                     else:
                         logger.log(
-                            u"Not updating episodes for show " + cur_show.name + " because it's last/next episode is not within the grace period.",
+                            u"Not updating episodes for show {0} because it's last/next episode is not within the grace period.".format(cur_show.name),
                             logger.DEBUG)
-                        # pi_list.append(sickbeard.showQueueScheduler.action.refreshShow(cur_show, True))
             except (CantUpdateShowException, CantRefreshShowException) as e:
-                logger.log(u"Automatic update failed: " + ex(e), logger.ERROR)
+                logger.log(u"Automatic update failed: {0}".format(str(e)), logger.ERROR)
```

Comment:
Format() already casts to string. You should use this: logger.log(u"Automatic update failed: {0}".format(ex(e)), logger.ERROR)